src/seeders/data/2-products/products.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "_id": { "$oid": "5e8508ea69a10e42d07414bd" },
+        "name": "Marshall Stanmore II - Alta Voz Bluetooth, Color Negro, EU",
+        "description": "El altavoz stanmore ii es el más versátil de la línea marshall y, además, es perfecto para cualquier habitación, ya sea grande o pequeña",
+        "imageUrl": ["https://images-na.ssl-images-amazon.com/images/I/71VfR7Xhk8L._AC_SL1000_.jpg"],
+        "createdAt": "4/06/2021",
+        "updatedAt": "5/06/2021",
+        "owner": "Federico",
+        "categoryId": "6543",
+        "userFavs": []
+    },
+    {
+        "_id": { "$oid": "5e850bd669a10e42d07414c1" },
+        "name": "Consola PlayStation 5 + Marvel´s Spider-Man Miles Morales + Destruction Allstars",
+        "description": "Incluye un mando DualSense y los juegos Marvel´s Spider-Man: Miles Morales y Destruction Allstars",
+        "imageUrl": ["https://images-na.ssl-images-amazon.com/images/I/61GVci%2B2CkS._AC_SL1500_.jpg"],
+        "createdAt": "4/06/2021",
+        "updatedAt": "5/06/2021",
+        "owner": "Eudald",
+        "categoryId": "2456",
```

Comment:
The category id should be the one used internally by mongo. If we don't have categories in the seeder, it's ok, but once we have them, we will have to change this.